Added Content Component to About Page and added some global CSS classes

ayushnayyar opened the pull request:
- Added about-content.json to assets directory
- Made the Content component reusable, by accepting props (Content.jsx)
- Added Content Component to About Page and supplied relevant pro (about.jsx)
-Added few CSS classes for global styles in main.scss

#22

ayushnayyar committed:
Added Content Component to About Page and added some global CSS classes

- Added about-content.json to assets directory
- Made the Content component reusable, by accepting props (Content.jsx)
- Added Content Component to About Page and supplied relevant pro (about.jsx)
-Added few CSS classes for global styles in main.scss

pruthvi2103 commented on src/pages/about.jsx:
```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import React from 'react';
+import aboutPageContent from '../assets/about-content.json';
+import Content from '../components/Content';
 
 const About = () => {
-	return <div></div>;
+    const content = aboutPageContent.map((aboutContent) => {
+        return (
+            <div>
+                <Content
+                    headline={aboutContent.headline}
+                    content={aboutContent.content}
+                    iframeTitle={aboutContent.tedxIframeTitle}
+                    iframeLink={aboutContent.tedxIframeLink}
+                />
+            </div>
+        );
+    });
+
+    return <div>{content}</div>;
```
I see a lot of 'content content' which is not making the code readable to an external reader kindly do something like naming the map function const's name to `renderContent`  (referring to lines `6`& `19`)

pruthvi2103 commented on src/pages/about.jsx:
```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import React from 'react';
+import aboutPageContent from '../assets/about-content.json';
```
Try to use significant keywords to refer to data it would be great if we name this as 
```
import aboutPageData from ...
```
it quickly helps us in knowing if variables are components,data,functions or actions

pruthvi2103 commented on src/components/content.jsx:
```diff
@@ -1,33 +1,21 @@
-                            local, self-organized events that bring people
-                            together to share a TED-like experience. Our event
-                            is called TEDxJNEC, where x = independently
-                            organized TED event. At our TEDxJNEC event, TEDTalks
-                            video and live speakers will combine to spark deep
-                            discussion and connection in a small group. The TED
-                            Conference provides general guidance for the TEDx
-                            program, but individual TEDx events, including ours,
-                            are self-organized.
-                        </p>
+                        <h1 className='content-headline'>{headline}</h1>
+                        <p>{content}</p>
                     </div>
                     <div className='column'>
                         <div className='content-wrapper'>
                             <iframe
                                 className='tedx-iframe'
-                                title='TEDx'
-                                src='https://www.youtube.com/embed/d0NHOpeczUU'
+                                title={iFrameTitle}
+                                src={iframeLink}
                                 frameborder='0'
                                 allow='accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture'
                                 allowfullscreen
```
Consider this as an optional task:

Can we make this component further reusable by giving it a prop to state if we want the iframe to the left and content to right or vice-versa?

purely an optional task and if you are interested kindly ask for @sohamtamboli 's help

pruthvi2103 reviewed (changes requested):
@ayushnayyar Kindly adress these issues,

Thanks

ayushnayyar committed:
Fixed Errors related to iFrame and Map Lists, and changed a few variable names

- Changed aboutPageContent to aboutPageData ,aboutContent to aboutData and changed content to renderContent in about.jsx.
- Added reversing order functionality in Content component (Content.jsx) which can be configured as a prop.

ayushnayyar commented:
Fixed Errors related to iFrame and Map Lists, and changed a few variable names

- Changed aboutPageContent to aboutPageData ,aboutContent to aboutData and changed content to renderContent in about.jsx.
- Added reversing order functionality in Content component (Content.jsx) which can be configured as a prop.

pruthvi2103 reviewed (approved):
The code looks way better!! Nice job! there is an alternative way of doing the revert thing using CSS classes but what you did makes more sense!

pruthvi2103 commented:
@ayushnayyar  small feedback, tasks such as these should be done in 2 PRs 1 which addressed the About page and the other which addressed the CSS classes. Helps us in case we want to revert just the CSS or the about page commits lets follow this from next time.
